stack.yaml

```diff
@@ -5,7 +5,9 @@ packages:
 
 docker:
   enable: false
-  repo: fpco/stack-build:lts-14.27
+
+  #image: fpco/alpine-haskell-stack:8.6.5
+  image: fpco/alpine-haskell-stack@sha256:49e7e15f3b1d3f882ba5bb701463b1d508fbf40e5aafce6ea31acd210da570ba
```

Comment:
Is the intent that `stack --docker` _always_ builds using Alpine?  If so, would that mean we'd always have to remember to pass in extra flags along with `--docker` (e.g. `--flag=stack:static --system-ghc`)?

An alternative would be to leave the image here as it was, and have the release process pass in `--docker-image=fpco/alpine-haskell-stack@sha256:49e7e15f3b1d3f882ba5bb701463b1d508fbf40e5aafce6ea31acd210da570ba`.

Reply:
My concern with that approach is that, then, we have to specify the Alpine image ID in multiple places. I'd like to leave it this way for now. I doubt people are building Stack with the Ubuntu Docker images anyway. I'm investigating a significantly different strategy for building with Github Actions, and may be able to simplify things there and not incur the update-in-two-places problem.